Macbook-setup.md

```diff
@@ -0,0 +1,170 @@
+# Mac - new system setup instructions
+
+These instructions describe software installations for MacOS as of 6/24/2021 and are based on an Intel-based Mac running Big Sur (macOS v11.4). Any problems with this process may result from a different architecture, OS version, or newer versions of the software being installed.
+
+_Note: MacOS uses Z shell (zsh) as the default shell. If you prefer bash, you can make that the default shell under System Preferences > Users and Groups (unlock the lock icon to make changes) > right click your user, select 'Advanced Options' and change 'Login shell' to /bin/bash. Then use ~/.bashrc instead of ~/.zshrc throughout this setup._
+
+## Download and install the following software:
+ - [Slack](https://slack.com/downloads/mac)
+ - [Zoom](https://zoom.us/download)
+ - [OneDrive](https://www.microsoft.com/en-us/microsoft-365/onedrive/download)*
+ - Web browser, such as [Chrome](https://www.google.com/chrome/)*
+ - [LastPass browser extension](https://lastpass.com/misc_download2.php)
+ - Text editor, such as [VSCode](https://code.visualstudio.com/download)
+ - [AWS CLI v2](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2-mac.html)
+ - [Postgres.app](https://postgresapp.com/downloads.html) (get "Postgres.app with all currently supported versions" - we are currently running v10.x in production)
+ - [pgAdmin4](https://www.pgadmin.org/download/)
+
+\* _may already be installed by UW IT_
+
+---
+## Install Homebrew 
+
+Follow the Homebrew installation instructions [here](https://docs.brew.sh/Installation).
+
+---
+## Install Python 3.6.9 via pyenv
+
+At time of writing, MacOS ships with Python 2.7.16 (/usr/bin/python) and Python 3.8.2 (/usr/bin/python3). We will leave these installed but will not use them, instead opting for pyenv to manage python versions (starting with Python 3.6.9 to match current production server).
+```
+brew install zlib sqlite bzip2 libiconv libzip xz pyenv 
+LDFLAGS="-L$(brew --prefix zlib)/lib -L$(brew --prefix bzip2)/lib" CPPFLAGS="-I$(brew --prefix xz)/include" pyenv install --patch 3.6.9 < <(curl -sSL https://github.com/python/cpython/commit/8ea6353.patch\?full_index\=1)
```

Comment:
Awesome that you were able to find an official patch for the Python build issues we were having!